### PR TITLE
Add some clarifications

### PR DESCRIPTION
**Pull request by joseluisbolos**

### CODE REVIEW

File: practices/dependency-management.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ Specifying the versions in this way ensures that:
 
 Don't use wildcards (i.e. `*`, `x`) in the package numbers, or keywords like `latest`. Not only they make the version numbers harder to read but they also make it harder to predict what exact versions will be installed.
 
+If you use `npm install --save` a lot, you may want to change npm's config so it uses tilde by default when saving dependencies:
+
+```sh
+npm config set save-prefix '~'
```
Comment (danyalaytekin) [inline practices/dependency-management.md]: Is this an alternative to the version given in the docs, `npm config set save-prefix='~'`?  https://docs.npmjs.com/misc/config

Comment (joseluisbolos) [inline practices/dependency-management.md]: Apparently both versions are acceptable, according to their docs. Do we have any preference?

Comment (danyalaytekin) [inline practices/dependency-management.md]: Na, go ahead with your preference

Comment (joseluisbolos) [inline practices/dependency-management.md]: Awesome 👍

**Pull request merged**